core/src/test/scala/github/ParseGithubOpenApiTest.scala

```diff
@@ -0,0 +1,16 @@
+package net.zhenglai
+package github
+
+import io.swagger.v3.parser.OpenAPIV3Parser
+import org.scalatest.FunSuite
+
+
+class ParseGithubOpenApiTest extends  FunSuite{
+
+  test("parse github open api"){
+    val openApi = new OpenAPIV3Parser().read("ghes-3.1.json")
```

Comment:
you need take care of the working directory. I guess you need join theresources dir and the ghes-3.1.json to get full correct path